core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -814,6 +879,49 @@ public void close() {
     }
   }
 
+  /**
+   * Selects the MasterWorkerInfo according to addresses from report capacity command input.
```

Comment:
"from report capacity command input." is really about the caller information, this can change and evolve in the future. I wouldn't put it in this javadoc.
Also can you make the method javadoc more specific ? Is it basically "selects MasterWorkerInfo whose address is in the given addresses"?

Reply:
Changed to
```java
  /**
   * Selects the MasterWorkerInfo whose host or related IP address exists in addresses.
   *
   * @param addresses the address set that user passed in
```